frontend/packages/topology/integration-tests/support/pages/topology/topology-page.ts

```diff
@@ -76,7 +76,9 @@ export const topologyPage = {
     });
   },
   verifyToplogyPageNotEmpty: () => {
-    cy.get('*[class="odc-topology"]').then(($el) => {
+    // Wait for topology page to load completely - increased timeout for yup async validation
+    cy.get('[class*="odc-topology"]', { timeout: 60000 }).should('be.visible');
```

Comment:
Additionally, why the use of wildcard in `[class*="odc-topology"]`, which matches any element where class attribute contains "odc-topology". Not recommended by CY and could be fragile. Maybe add `data-test` attribute instead.